src/mon/Elector.cc

```diff
@@ -496,7 +506,8 @@ void Elector::ping_check(int peer)
     // Monitor no longer exists in the monmap,
     // therefore, we shouldn't ping this monitor
     // since we cannot lookup the address!
-    dout(20) << __func__ << "peer >= ranks_size" << dendl;
+    dout(5) << __func__ << "peer >= ranks_size ... droping to prevent "
```

Comment:
s/droping/dropping/

Do we still want to keep this at 5? I guess it's not a common operation so no harm.